contracts/test/v0.8/KeeperRegistry.test.ts

```diff
@@ -966,7 +966,12 @@ describe('KeeperRegistry', () => {
       it('uses the fallback gas price if the feed price is non-sensical [ @skip-coverage ]', async () => {
         const normalAmount = await getPerformPaymentAmount()
         const roundId = 99
-        const updatedAt = Math.floor(Date.now() / 1000)
+
+        const currentBlockNum = await ethers.provider.getBlockNumber()
+        const currentBlock = await ethers.provider.getBlock(currentBlockNum)
+        const currentTimestamp = currentBlock.timestamp
```

Comment:
Do we need `currentTimestamp`? Can we just define `const updatedAt = currentBlock.timestamp`?

The same goes for the other two changes.

Reply:
done